### ISSUE TITLE
Header does not work

### ISSUE THREAD

**Issue by ninpl** [digitalcraftsman/hugo-type-theme#18]
When you generate the site, even if you configure config.toml, it does not create the following lines:

```
<div class="call-out" style="background-image: url('https://domain//img/bg.png')">
	textHeader
</div>
```

You need to manually enter it into index.html

**Comment by andygrunwald** [comment]
#20 will fix this

**Comment by ninpl** [comment]
Thanks @andygrunwald

**Comment by digitalcraftsman** [comment]
Fixed by #20.
